travis-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+
+dotnet test -c Debug -f netcoreapp1.0 MoreLinq.Test
```

Comment:
Isn't this a duplicate of the 3rd instruction (`dotnet test -f netcoreapp1.0 MoreLinq.Test`) under `script:` section of `.travis.yml'?